internal/report.go

```diff
@@ -355,9 +357,12 @@ func ok(total, badCount int64) bool {
 	return badCount < total/3
 }
 
-func rateConversion(rows, badRows, cols, warnings int64, missingPKey, summary bool) string {
-	return fmt.Sprintf("Schema conversion: %s.\n", rateSchema(cols, warnings, missingPKey, summary)) +
-		fmt.Sprintf("Data conversion: %s.\n", rateData(rows, badRows))
+func rateConversion(rows, badRows, cols, warnings int64, missingPKey, summary bool, schemaOnly bool) string {
```

Comment:
I'm fine with this at the moment, but just noting that it'll probably need to change when we add support for --data-only.

The use of conv.SchemaMode here is a bit fragile here (in the arg passed to rateConversion) since it reflects the last mode used, not how the tool was configured to run.

For example, if we do both schema and data mode, it will be data mode.
If we just do data mode, it will be data mode.
If we just to schema mode, it will schema mode.

Longer term, I think what we really want is to pass in how the tool was configured.
If we did schema processing, we should print schema processing stats.
If we did data processing, we should print data processing stats.

Reply:
As discussed, we'd like to generate a full report (schema and data stats) when we run in data mode.

Moreover, we always have to do data processing last, so even though conv.SchemaMode reflects the last mode we ran int, it always tells us whether we did data processing.

In short, let's leave this as is.